.github/workflows/build.yml

```diff
@@ -0,0 +1,89 @@
+name: Build and Upload pluto-logtool
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    # The first two permissions are needed to interact with GitHub's OIDC Token endpoint.
+    # The second set of two permissions are needed to write test results back to GH
+    permissions:
+      id-token: write
+      contents: read
+      checks: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "16.15.1"
+
+      # GITHUB_RUN_NUMBER always starts from 1, so to maintain compatibility we adjust it here
+      - name: Make GITHUB_RUN_NUMBER env var available outside of shells
+        working-directory: ${{env.GITHUB_WORKSPACE}}
+        shell: bash
+        run: |
+          echo GITHUB_RUN_NUMBER=$(( GITHUB_RUN_NUMBER + 200 ))>> $GITHUB_ENV
```

Comment:
Do we need the builds to start from 200 for any reason?

Reply:
The reason Andy gaves was 'maintain compatibility'.